include/envoy/server/guarddog.h

```diff
@@ -1,7 +1,10 @@
 #pragma once
 
 #include "envoy/common/pure.h"
+#include "envoy/common/time.h"
+#include "envoy/config/bootstrap/v3/bootstrap.pb.h"
 #include "envoy/server/watchdog.h"
+#include "envoy/thread/thread.h"
```

Comment:
nit: Are envoy/config/bootstrap/v3/bootstrap.pb.h and/or envoy/common/time.h used by this header?

Reply:
Good catch, must have happened at some point in development (but it's unnecessary)